dart/dynamics/detail/MultiDofJoint.h

```diff
@@ -37,6 +37,19 @@
 #ifndef DART_DYNAMICS_DETAIL_MULTIDOFJOINT_H_
 #define DART_DYNAMICS_DETAIL_MULTIDOFJOINT_H_
 
+#define MULTIDOFJOINT_REPORT_DIM_MISMATCH( func, arg )              \
+  dterr << "[MultiDofJoint::" #func "] mismatch beteween size of "  \
+        << #arg " [" << arg .size() << "] and the number of "       \
+        << "DOFs [" << getNumDofs() << "] for Joint named ["        \
+        << getName() << "]\n";                                      \
```

Comment:
I'm never really sure when to use periods in warning messages. I think usually if there is only one sentence for the whole message, I don't bother with a period, but I'll include periods if there are multiple sentences in the message.

That said, I have nothing against putting a period at the end.